test/api_integration/apis/index_patterns/fields_for_time_pattern_route/query_params.js

```diff
@@ -27,16 +27,26 @@ export default function ({ getService }) {
     before(() => esArchiver.load('index_patterns/daily_index'));
     after(() => esArchiver.unload('index_patterns/daily_index'));
 
-    it('requires `pattern` and `look_back` query params', () =>
+    it('requires `pattern` query param', () =>
       supertest
         .get('/api/index_patterns/_fields_for_time_pattern')
-        .query({ pattern: null })
+        .query({ look_back: 1 })
         .expect(400)
         .then(resp => {
-          expect(resp.body.validation).to.eql({
-            keys: ['pattern', 'look_back'],
-            source: 'query',
-          });
+          expect(resp.body.message).to.contain(
+            '[request query.pattern]: expected value of type [string] but got [undefined]'
```

Comment:
`@kbn/config-schema` doesn't return the missing required field. 

So, I broke the test into 2 separate tests.